RRemoteConfig/URLRequest+Headers.swift

```diff
@@ -0,0 +1,17 @@
+extension URLRequest {
+    mutating func setExtraHeaders(from environment: Environment) {
```

Comment:
I don't see any tests for this. Is it possible to test that these headers are getting added?

Reply:
I added tests for the headers.